src/Components/Artwork/Details/__tests__/Details.jest.tsx

```diff
@@ -636,6 +636,28 @@ describe("Details", () => {
 
       expect(html).toContain("Increased Interest")
     })
+
+    it("does not show the number of bids when there are bids on the sale artwork but no auction signals", async () => {
+      const data: any = {
+        ...artworkInAuction,
+        collectorSignals: {
+          ...artworkInAuction?.collectorSignals,
+          auction: null,
```

Comment:
This reproduces the state where an artwork has no auction signal (e.g. after the lot ends), [but...](https://github.com/artsy/force/pull/14613/files#r1786984886)